openshift_sti_images/ruby.adoc

```diff
@@ -111,10 +110,25 @@ By omitting the `VERSION` parameter, the build/test action will be performed on
 * `hack/` - Folder contains scripts which are responsible for build and test actions performed by the `Makefile`.
 * `Makefile` - Make utility for simplifying image build and test actions.
 
+== Image name structure
+
+*Structure: openshift/`1`-`2`-`3`*
```

Comment:
Just to be clear - I dropped that into a code block to show the raw AsciiDoc that I'm suggesting you use.